.ci/pytorch/python_doc_push_script.sh

```diff
@@ -105,7 +105,12 @@ if [ "$is_main_doc" = true ]; then
     echo undocumented objects found:
     cat build/coverage/python.txt
     echo "Make sure you've updated relevant .rsts in docs/source!"
-    exit 1
+    # NB: Due to some duplication of the following modules/functions, we keep
+    # them as expected failures for the time being instead of return 1
+    # - torch.nn.utils.weight_norm
+    # - torch.nn.utils.spectral_norm
+    # - torch.nn.parallel.data_parallel
+    # - torch.ao.quantization.quantize
```

Comment:
I think you should change docs/conf.py to ignore these 4, it looks like this part of the coverage check is implemented ourself

Reply:
I tried, but none of the `coverage_ignore_` works AFAICT https://www.sphinx-doc.org/en/master/usage/extensions/coverage.html like we need.  `make coverage` finishes successfully, but it always prints these out as a warning and the script treats that as extra failures.  This is not ideal, but is a low risk change IMO to keep trunk sane for now.  I plan to land this, then try to upgrade sphinx to a newer version to see if it helps.

Reply:
https://github.com/pytorch/pytorch/blob/18c9d460682293bae23386e9bc40932eee1d8add/docs/source/conf.py#L3596-L3619

Reply:
Ohhh, got it.  TIL, the script is the one responsible for that.  So, let me push a follow-up fix.